### PR TITLE
Switch jQuery example to new UI

### PR DESCRIPTION
**Pull request by arthurvr**
See https://github.com/tastejs/todomvc/issues/1110. Switches the jQuery example over to `todomvc-app-css`, and thus, the new UI.

### CODE REVIEW

File: examples/jquery/bower.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "todomvc-jquery",
   "dependencies": {
-    "todomvc-common": "~0.3.0",
+    "todomvc-common": "~1.0.1",
+    "todomvc-app-css": "~1.0.1",
     "jquery": "~2.1.1",
     "handlebars": "~2.0.0",
     "director": "~1.2.2"
```
Comment (sindresorhus) [inline examples/jquery/bower.json]: Can you switch it to npm?

Comment (sindresorhus) [inline examples/jquery/bower.json]: Can you also add a .gitignore as seen here: https://github.com/tastejs/todomvc/pull/1111/files#diff-c39d5a21f6bb2f461f2549116f937d95R1

Comment (arthurvr) [inline examples/jquery/bower.json]: @sindresorhus Both are done.

Comment (arthurvr) [inline examples/jquery/bower.json]: Btw, that wasn't done in https://github.com/tastejs/todomvc/commit/9c6599c19aabb813ec291fd79fafe3ff83210f54 too.

**Comment by addyosmani** [comment]
Thanks for helping power through these, @arthurvr :)

File: examples/jquery/bower.json (hunk 1 above)
Comment (sindresorhus) [inline examples/jquery/bower.json]: Yes, I know. Not a requirement, but nice to do it when we change around things anyways. Thanks for doing it btw :)

Comment (arthurvr) [inline examples/jquery/bower.json]: Ready to merge, then?

**Comment by passy** [comment]
Awesomesauce! Thanks, @arthurvr! :+1:

**Pull request merged**